_posts/2014-07-07-mailpile.md

```diff
@@ -8,7 +8,7 @@ authorUrl: https://github.com/jparyani
 Hello, I'm Jason, a new member of the Sandstorm team. Today I'm excited to
 announce that I've extended Sandstorm to support e-mail, and ported
 [Mailpile Alpha II](https://www.mailpile.is/) to demonstrate. Mailpile is
-available on the [Sandstorm app list](https://sandstorm.io/apps/) right now, so
+available on the [Sandstorm app list](https://apps.sandstorm.io) right now, so
```

Comment:
I would point this at sandstorm.org/apps for now, as Mailpile isn't available on the App Market.